libraries/lib-module-manager/PluginManager.cpp

```diff
@@ -373,14 +373,7 @@ void PluginManager::Initialize(FileConfigFactory factory)
       // Allow the module to auto-register children
       module->AutoRegisterPlugins(*this);
    }
-
-   // And finally check for updates
-#ifndef EXPERIMENTAL_EFFECT_MANAGEMENT
    CheckForUpdates();
-#else
-   const bool kFast = true;
-   CheckForUpdates( kFast );
```

Comment:
To make this commit have no behavior changes, the other conditional compilation branch should be kept.

Reply:
Thanks!